package.json

```diff
@@ -22,18 +22,14 @@
   "license": "ISC",
   "homepage": "https://bitbucket.org/gtmhub/webapp-plugins-sdk#readme",
   "devDependencies": {
-    "@babel/cli": "^7.15.7",
-    "@babel/core": "^7.15.5",
-    "@babel/preset-typescript": "^7.15.0",
     "@types/cors": "^2.8.12",
     "@types/express": "^4.17.13",
-    "babel-core": "^7.0.0-bridge.0",
     "concurrently": "^6.3.0",
     "cors": "^2.8.5",
     "express": "^4.17.1",
-    "nodemon": "^2.0.12",
     "ts-loader": "^9.2.6",
     "ts-node": "^10.2.1",
+    "ts-node-dev": "^2.0.0",
```

Comment:
What does this do compared to `ts-node`?

Reply:
It's quiet more performant for development compared to ts-node, here's better explanatory of it.

`It restarts target node process when any of required files changes (as standard node-dev) but shares [Typescript](https://github.com/Microsoft/TypeScript/) compilation process between restarts. This significantly increases speed of restarting comparing to node-dev -r ts-node/register ..., nodemon -x ts-node ... variations because there is no need to instantiate ts-node compilation each time.`
[](https://www.npmjs.com/package/ts-node-dev)

Reply:
Do you need `ts-node` then? Can we remove it?